Source: https://mail-archives.apache.org/mod_mbox/mesos-reviews/201610.mbox/%3C20161014170728.1720.54446@reviews.apache.org%3E
Upstream: No
Reason: fixes compilation with musl that doesn't define W* macros.

--- a/src/widget.cc
+++ b/src/widget.cc
@@ -31,6 +31,10 @@
 #include "vteptyinternal.hh"
 #include "debug.h"
 
+#ifndef W_EXITCODE
+#define W_EXITCODE(ret, sig)  ((ret) << 8 | (sig))
+#endif
+
 using namespace std::literals;
 
 namespace vte {
